docs/terminal/sydney.md

```diff
@@ -348,15 +353,15 @@ It is vital that Approach controllers ensure all arriving aircraft are establish
 ##### MARLN Corridor
 Aircraft are permitted to cross the MARLN corridor at or above `A060` without coordination with APP. DEP is responsible for separation with respect to aircraft in the corridor.
 
-#### SDN / SFW/SAN
+#### SDN / SFW / SAN
```

Comment:
ive been trying to use the spaced slashes and the non-spaced slashes to distinguish that its coordination between:
SDN; and
SFW/SAN

as opposed to coordination between SDN, SFW, and SAN

Reply:
Ah right, sorry.  I thought you might have just missed the formatting.  Maybe we should add '&' or another character to differentiate.  Like `SDN & SFW/SAN` or something?

Reply:
Or something like this?
![image](https://user-images.githubusercontent.com/41880668/222609978-b2679a51-a59d-4c42-a71e-f8c1322a5cac.png)

Reply:
yeah actually i prefer this, nice. long term i can work on changing all the other pages to match this

Reply:
Too easy, I'll update SY TMA with that now